### PR TITLE
Translate enforcement docs to all non-rust languages

### PR DESCRIPTION
**Pull request by gkaemmer**

### CODE REVIEW

File: docs/content/any/guides/enforcement/resource/index.md [hunk 1]
```diff
@@ -33,43 +23,32 @@ If you only perform one type of authorization in your app, it should be
...
 <!-- You'll see this method in a lot of our guides and examples, because it's the
 simplest way to use Oso in your app. -->
 
-```python
-oso.authorize(user, "approve", expense)
-```
+{{% exampleGet "exampleCall" %}}
 
-The `authorize` method checks all of the `allow` rules in your policy, and
+The `{{< exampleGet "authorize" >}}` method checks all of the `allow` rules in your policy, and
```
Comment (gj) [inline docs/content/any/guides/enforcement/resource/index.md:40]: comma splice

```suggestion
The `{{< exampleGet "authorize" >}}` method checks all of the `allow` rules in your policy and
```

File: docs/content/any/guides/enforcement/resource/index.md [hunk 2]
```diff
@@ -33,43 +23,32 @@ If you only perform one type of authorization in your app, it should be
...
 ensures that there is an `allow` rule for that applies to the given user,
```
Comment (gj) [inline docs/content/any/guides/enforcement/resource/index.md:41]:
```suggestion
ensures that there is an `allow` rule that applies to the given user,
```

File: docs/content/any/guides/enforcement/resource/data/data.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+exampleCall: |-
+    ```python
+    oso.authorize(user, "approve", expense)
+    ```
+getExpense: |-
+    ```python
+    def get_expense(user, expense_id):
+        expense = db.fetch(
+            "SELECT * FROM expenses WHERE id = %", expense_id)
+        oso.authorize(user, "read", expense)
```
Comment (gj) [inline docs/content/any/guides/enforcement/resource/data/data.md:24]: should this be `"approve"` to match the `authorize` call & `allow` rule above it?

```suggestion
        oso.authorize(user, "approve", expense)
```

image

Comment (gkaemmer) [reply docs/content/any/guides/enforcement/resource/data/data.md:24]: Ah yeah the method should probably be approveExpense, rats

Comment (gkaemmer) [reply docs/content/any/guides/enforcement/resource/data/data.md:24]: Will update everywhere

File: docs/content/any/guides/enforcement/_index.node.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+title: Enforcing an Oso Policy (Oso 0.20.0 Beta)
```
Comment (dhatch) [inline docs/content/any/guides/enforcement/_index.node.md:2]: titles need update

**Review by dhatch — COMMENTED** [review]
We need to add something that says rust guide coming soon.

- Code snippets on java field page are messed up: http://localhost:1313/java/guides/enforcement/field.html

**Review by gj — APPROVED** [review]

**Pull request merged**